rest/model/host.go

```diff
@@ -36,8 +37,9 @@ type APIHost struct {
 	AttachedVolumeIDs     []string    `json:"attached_volume_ids"`
 }
 
-// HostPostRequest is a struct that holds the format of a POST request to /hosts
-// the yaml tags are used by hostCreate() when parsing the params from a file
+// HostRequestOptions is a struct that holds the format of a POST request to
+// /hosts the yaml tags are used by hostCreate() when parsing the params from a
```

Comment:
I am confused by what this is saying?

Reply:
I didn't write this comment, and I have no idea what it's saying.

Reply:
:)